src/rgw/rgw_acl_s3.cc

```diff
@@ -18,9 +18,11 @@ using namespace std;
 
 #define RGW_URI_ALL_USERS	"http://acs.amazonaws.com/groups/global/AllUsers"
 #define RGW_URI_AUTH_USERS	"http://acs.amazonaws.com/groups/global/AuthenticatedUsers"
+#define RGW_URI_LOG_DELIVERY    "http://acs.amazonaws.com/groups/s3/LogDelivery" //TODO cleanup redundant define
```

Comment:
@mikulely http://acs.amazonaws.com/groups/s3/LogDelivery
This site can’t be reached